### PR TITLE
Extnet name instead of id for examplesetup and getstarted

### PR DESCRIPTION
**Pull request by dschwabeS11**

### CODE REVIEW

File: example-setup/clustersetup-env-testing-small.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # This parameter file is configured to be used with a 10 Core / 40 GB RAM quota provided by SysEleven GmbH.
 parameters:
   # The following public network ID belongs to the SysEleven Stack region 'cbk' and should be changed if using other regions.
```
Comment (stefreak) [inline example-setup/clustersetup-env-testing-small.yaml]: fix outdated comment

File: example-setup/clustersetup-env.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # This parameter file is configured to be used with a 30 Core / 120 GB RAM quota provided by SysEleven GmbH.
 parameters:
   # The following public network ID belongs to the SysEleven Stack region 'cbk' and should be changed if using other regions.
```
Comment (stefreak) [inline example-setup/clustersetup-env.yaml]: Fix outdated comment

File: example-setup/scripts/install_generic.sh [hunk 3]
```diff
@@ -35,8 +35,7 @@ cat <<EOF> /etc/consul.d/consul.json
   "bootstrap_expect": 3,
   "server": true,
   "enable_script_checks": true,
-  "disable_remote_exec": true,
-  "start_join": ["192.168.2.11", "192.168.2.12", "192.168.2.13"]
```
Comment (stefreak) [inline example-setup/scripts/install_generic.sh]: its ok this time, but this seems to be an unrelated change. please keep unrelated changes separate from each other :)

File: gettingStarted/sysElevenStackKickstart.yaml [hunk 4]
```diff
@@ -64,7 +64,8 @@ resources:
               mkdir /home/syseleven/.ssh
               curl -s http://169.254.169.254/latest/meta-data/public-keys/0/openssh-key > /home/syseleven/.ssh/authorized_keys
               cat <<EOF> /home/syseleven/.ssh/config
-              Host 185.56.* 
+              # Accept all S11 FloatingIPs HostKeys
```
Comment (stefreak) [inline gettingStarted/sysElevenStackKickstart.yaml]: Why are you doing this? This is insecure and should not be done. This is basically disabling SSHs man in the middle protection completely.

Also this is an unrelated change. Please do not put many unrelated changes in one commit, and defenitely not in one merge request!

File: gettingStarted/sysElevenStackKickstart.yaml [hunk 5]
```diff
@@ -101,7 +102,7 @@ resources:
          - content: |
               #!/usr/bin/env bash
               until id syseleven > /dev/null; do sleep 2; done
-              url="https://releases.hashicorp.com/terraform/0.10.2/terraform_0.10.2_linux_amd64.zip"
+              url="https://releases.hashicorp.com/terraform/0.11.1/terraform_0.11.1_linux_amd64.zip"
```
Comment (stefreak) [inline gettingStarted/sysElevenStackKickstart.yaml]: unrelated change

**Comment by dschwabeS11** [comment]
Thanks for checking. I checked the wrong branch out before. Will do it again but properly this time.

**Comment by stefreak** [comment]
@dschwabeS11 you can use git cherry-pick for the commits you want to keep.
